soccer/gameplay/plays/offense/Offenseplay1.py

```diff
@@ -0,0 +1,101 @@
+import robocup
+import standard_play
+import behavior
+import constants
+import main
+import skills.move
+import skills.capture
+import enum
+import evaluation
+import tactics.coordinated_pass
+import tactics.defense
+import play
+
+class Offenseplay(play.Play):
+	class State(enum.Enum):
+
+		setup = 1
+		passing = 2
+		shooting = 3
+	def __init__(self):
+		super().__init__(continuous=True)
+
+		self.add_state(Offenseplay.State.setup,
+						behavior.Behavior.State.running)
+		self.add_state(Offenseplay.State.passing,
+						behavior.Behavior.State.running)
+		self.add_state(Offenseplay.State.shooting,
+						behavior.Behavior.State.running)
+
+		self.add_transition(behavior.Behavior.State.start,
+						Offenseplay.State.setup, lambda: True, 'immediately')
+		self.add_transition(Offenseplay.State.setup,
+                            Offenseplay.State.passing,
+                            lambda: self.all_subbehaviors_completed(),
+                            'moving completed')
+		self.add_transition(Offenseplay.State.passing,
+                            Offenseplay.State.shooting,
+                            lambda: self.all_subbehaviors_completed(),
+                            'passing completed')
+		self.add_transition(Offenseplay.State.shooting, 
+							behavior.Behavior.State.completed,
+            				lambda: self.all_subbehaviors_completed(),
+            				'Kick completed')
+		self.r1 = robocup.Point(1,5)
+		self.r2 = robocup.Point(-1,5)
```

Comment:
These should be percent of field size so we can run on any field. They should also be a little more descriptive in name. I have no idea what they represent just glancing at the code